src/Core/src/Hosting/AppHostBuilderExtensions.cs

```diff
@@ -101,6 +102,22 @@ public static IAppHostBuilder UseMauiServiceProviderFactory(this IAppHostBuilder
 			return builder;
 		}
 
+		public static IAppHostBuilder UseMicrosoftExtensionsServiceProviderFactory(this IAppHostBuilder builder)
```

Comment:
@PureWeen / @Redth / @rmarinho - any problem with adding this extension method? Hopefully we figure out a way to avoid this entirely, but for preview4, how's this?